dice_ml/explainer_interfaces/dice_pytorch.py

```diff
@@ -75,6 +75,7 @@ def generate_counterfactuals(self, query_instance, total_CFs, desired_class="opp
         :param tie_random: Used in rounding off CFs and intermediate projection.
         :param stopping_threshold: Minimum threshold for counterfactuals target class probability.
         :param posthoc_sparsity_param: Parameter for the post-hoc operation on continuous features to enhance sparsity.
+        :param posthoc_sparsity_algorithm: Perform either linear or binary search. Takes "linear" or "binary".
```

Comment:
Say more---Binary search is only applicable when the feature shares a monotonic relationship with predicted outcome in the model.